silk/sql.py

```diff
@@ -77,7 +83,7 @@ def execute_sql(self, *args, **kwargs):
             return iter([])
         else:
             return
-    sql_query = q % tuple(force_str(param) for param in params)
```

Comment:
See #617 .  Let's keep this as is.

Reply:
I think #658 makes sense.  It's possible it might result in invalid SQL but given this SQL is meant to be shown for debugging, I think a human can make that decision rather than just triggering an exception.